web/crux-ui/locales/en/images.json

```diff
@@ -19,5 +20,8 @@
   "errors": {
     "preconditionFailed": "Node is unreachable"
   },
-  "unknownTag": "unknown"
+  "unknownTag": "unknown",
+  "invalidImageFormat": "Invalid format, please use 'Name[:Tag]'",
```

Comment:
Why are using `[]`?

Reply:
I wanted to show somehow that is optional, I'll need to ask @gerimate about some texts (this and the explanation as well) :)

Reply:
Okay, I found that this is the official format. I'm thinking of using capital letters for NAME[:TAG]. What do you think?